darwin/XCSoar.xcodeproj/xcshareddata/xcschemes/XCSoar.xcscheme

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1620"
+   version = "2.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES"
+      buildArchitectures = "Automatic">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Run Script"
+               scriptText = "if [ &quot;$ACTION&quot; = &quot;clean&quot; ]; then&#10;  $WORKSPACE_PATH/../../clean.sh&#10;fi&#10;">
+            </ActionContent>
+         </ExecutionAction>
+      </PreActions>
+      <BuildActionEntries>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "5CBFD0922D742E1000F05795"
```

Comment:
Where does this number come from? Can this key be left undefined?

Reply:
It's automatically set by XCode and needs to be consistent with [darwin/XCSoar.xcodeproj/project.pbxproj](https://github.com/XCSoar/XCSoar/blob/a6150c97fc282b03f19124ac132d91d283274bd2/darwin/XCSoar.xcodeproj/project.pbxproj). I also don't like that the XCode project indexes every file, but I'm afraid that's just how XCode works.